packages/tldraw/src/lib/ui/hooks/clipboard/pasteUrl.ts

```diff
@@ -16,25 +15,6 @@ export async function pasteUrl(
 	point?: VecLike,
 	sources?: TLExternalContentSource[]
 ) {
-	// Lets see if its an image and we have CORS
-	try {
-		// skip this step if the url doesn't contain an image extension, treat it as a regular bookmark
-		if (new URL(url).pathname.match(/\.(png|jpe?g|gif|svg|webp)$/i)) {
```

Comment:
this is the main thing that was removed that only worked a small percentage of the time giving a flaky / weird user experience